CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+- Support for `zf` and `zd` commands, for folding and unfolding custom code blocks ([@elazarcoh](https://github.com/elazarcoh)).
```

Comment:
needs a rebase - this can go under a new `### Added` section

Reply:
Hmm the rebase messed it up... I'll see what I can do.

Reply:
Done.